### ISSUE TITLE
bake plugin doesn't generate tests/bootstrap.php

### ISSUE THREAD

**Issue by alysson-azevedo** [cakephp/bake#307]
Last year I reported the issue #248 and fixed it on #249.
Then #259 reverted it.

Now composer still ignores https://github.com/cakephp/bake/blob/master/src/Template/Bake/Plugin/tests/bootstrap.php.ctp.

**Comment by markstory** [comment]
Closing as there is a pull request open now.

**Comment by alysson-azevedo** [comment]
is there?
note that this issue here happens when you run "cake bake plugin SomePlugin" and the SomePlugin/tests/bootstrap.php not being baked.

**Comment by markstory** [comment]
I thought the connected pull request was to fix this, my mistake.

**Comment by sitedyno** [comment]
`src/Template/Bake/Plugin/tests/bootstrap.php.ctp` is not shipped in the archive due to `src/Template/Bake/Plugin/.gitattributes`

Could the gitattributes file be renamed in the repo and then named appropriately after it's baked?

**Comment by markstory** [comment]
Good idea @sitedyno I can fix-up the gitattributes file so it doesn't exclude the bootstrap template.
